message_definitions/v1.0/ras_a.xml

```diff
@@ -4,6 +4,30 @@
   <version>0</version>
   <dialect>0</dialect>
   <enums>
+    <enum name="MAV_AUTOPILOT">
+      <description>Micro air vehicle / autopilot classes. This identifies the individual model.</description>
+      <entry value="21" name="MAV_AUTOPILOT_SKYDIO">
+        <description>Skydio - https://www.skydio.com/</description>
+      </entry>
+      <entry value="22" name="MAV_AUTOPILOT_SHIELD_AI">
+        <description>Shield AI - https://shield.ai/</description>
+      </entry>
+    </enum>
+    <enum name="MAV_PROTOCOL_CAPABILITY" bitmask="true">
+      <description>Bitmask of (optional) autopilot capabilities (64 bit). If a bit is set, the autopilot supports this capability.</description>
+      <entry value="262144" name="MAV_PROTOCOL_CAPABILITY_INDOOR_OBSTACLE_AVOIDANCE">
+        <description>Autopilot is capable of indoor obstacle avoidance. The obstacle visualization range (i.e. omnidirectional or single direction) is not determined by this flag and is system specific.</description>
+      </entry>
+      <entry value="524288" name="MAV_PROTOCOL_CAPABILITY_OUTDOOR_OBSTACLE_AVOIDANCE">
+        <description>Autopilot is capable of outdoor obstacle avoidance. The obstacle visualization range (i.e. omnidirectional or single direction) is not determined by this flag and is system specific.</description>
+      </entry>
+      <entry value="1048576" name="MAV_PROTOCOL_CAPABILITY_VIO_AUTONOMOUS_NAVIGATION">
+        <description>Autopilot is capable of autonomous navigation using Visual-Inertial Odometry algorithms and/or subsystems.</description>
+      </entry>
+      <entry value="2097152" name="MAV_PROTOCOL_CAPABILITY_AUTONOMOUS_EXPLORATION">
+        <description>Autopilot is capable of autonomous exploration tasks in indoor/outdoor environments. This necessarily means that the vehicle is capable of autonomous navigation and obstacle avoidance.</description>
+      </entry>
```

Comment:
Does this essentially mean "robot supports Exploration microservice"?

Reply:
Yes we could say tha